tests/unit/lms/services/canvas_api/_basic_test.py

```diff
@@ -37,37 +37,42 @@ def test_sends_builds_the_right_url(
             **{
                 "method": "METHOD",
                 "path": "default_path/",
+                "timeout": sentinel.timeout,
                 "schema": Schema,
                 key: value,
             }
         )
 
         http_session.send.assert_called_once_with(
-            Any.request.with_url(expected_url), timeout=Any()
+            Any.request.with_url(expected_url), timeout=sentinel.timeout
         )
 
     @pytest.mark.parametrize("method", ("GET", "POST", "PATCH"))
     def test_send_uses_the_request_method(
         self, basic_client, Schema, method, http_session
     ):
-        basic_client.send(method, "path/", schema=Schema)
+        basic_client.send(method, "path/", schema=Schema, timeout=sentinel.timeout)
 
-        http_session.send.assert_called_once_with(Any.request(method), timeout=Any())
+        assert http_session.send.call_args[0][0] == Any.request(method)
```

Comment:
`assert_called_once_with(Any.request(method), timeout=Any())` was the wrong thing to use here. This test is only trying to test that the right HTTP method is passed to `http_session.send()` so it only cares that the first positional argument (`call_args[0][0]`) is any request object with the right `method`